Deploy-CAM.ps1

```diff
@@ -1754,7 +1754,8 @@ function New-ConnectionServiceDeployment() {
                         -ResourceGroupName $csRGName `
```

Comment:
Looks liek we're still using CS# for the deployment name. Do we want to change that to CN#?

Reply:
Yeah I didn't rename all the internal variables since that would be an even more massive change. or the keyvault secrets since that would hose upgrades. I feel we should make a backlog story about 'completing renaming connection service to connector' to deal with those things if that makes sense to you. I think the resource group naming is properly handled though - or did I miss something?